plugin-server/src/worker/ingestion/hooks.ts

```diff
@@ -237,7 +237,11 @@ export class HookCommander {
         this.statsd?.increment('webhook_firings')
     }
 
-    private async postRestHook(hook: Hook, event: PluginEvent, person: Person | undefined): Promise<void> {
+    private async postRestHook(
+        hook: Hook,
+        event: PluginEvent,
+        person: CachedPersonData | Person | undefined
```

Comment:
I'm assuming it's fine for the hooks to get more limited person info compered to before & that it can be different from buffer or non-buffer events what fields are included. Are the hooks used by customers do we now what they assume?

Reply:
You'll notice that both `CachedPersonData` and `Person` types can be passed down.

Essentially TypeScript would scream if we tried to access a value that's not present in both interfaces. That's the reason I added more properties to `CachedPersonData`, so it contains everything the webhook handling needs.

Reply:
I meant that about the payload where we add data (line 247 just below)

Reply:
Ah yes that's a good point - hadn't spotted it

Reply:
Checked and the info that won't be sent is irrelevant

Reply:
how did you check, asking so I know next time how to do it

Reply:
I just looked at what the fields are that `Person` has which `CachedPersonData` doesn't. And I made the call that they're useless. Also spoke to Michael who wrote the Zapier integration

```
is_user_id
is_identified
properties_last_updated_at
properties_last_operation
version
```